src/language-js/parse-postprocess.js

```diff
@@ -272,4 +277,36 @@ function includeShebang(ast, options) {
   }
 }
 
+function replaceNullElements(node, ast, options) {
```

Comment:
Please add  a comment that explain motivation to introduce `Null` node or link to this PR.

Reply:
Done.